MapboxSearch/sample/src/main/java/com/mapbox/search/sample/MainActivity.kt

```diff
@@ -178,7 +178,9 @@ class MainActivity : AppCompatActivity() {
             setSupportActionBar(this)
         }
 
-        val apiType = if (BuildConfig.ENABLE_SBS) {
+        val apiType = if (BuildConfig.ENABLE_SEARCH_BOX) {
+            ApiType.SEARCH_BOX
+        } else if (BuildConfig.ENABLE_SBS) {
             ApiType.SBS
         } else {
             ApiType.GEOCODING
```

Comment:
If we just use a constant, an enum method could be added to just get the ApiType.

Reply:
Refactored this part, please take another look